src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
```

Comment:
this should be fluid and not fixed, it can be removed

Reply:
deleted !

Reply:
![Screen Shot 2022-04-11 at 3 27 35 PM](https://user-images.githubusercontent.com/58893691/162843895-60ac00ac-edd3-4b4b-b360-5bbd7f7a6f61.png)

Reply:
deleted the FooterCard  Width: 100rem; now it is better !